tests/contract/knative-channel-kafka/README.md

```diff
@@ -0,0 +1,25 @@
+# Knative Kafka channel acceptance tests
+
+## Overview
+
+This project contains the acceptance tests for the Knative Kafka channel https://github.com/kyma-incubator/knative-kafka. It can run as part of the Kyma testing process. The tests are written in Go. Run them as standard Go tests.
```

Comment:
```suggestion
This project contains the acceptance tests for the [Knative Kafka channel](https://github.com/kyma-incubator/knative-kafka). It can run as part of the Kyma testing process. The tests are written in Go, so you can run them as standard Go tests.
```